brave/src/test/java/brave/features/opentracing/OpenTracingAdapterTest.java

```diff
@@ -37,12 +36,12 @@
  * the core concepts.
  */
 public class OpenTracingAdapterTest {
+  static final ExtraField EXTRA_FIELD = ExtraField.newBuilder("user-id").build();
+
   List<zipkin2.Span> spans = new ArrayList<>();
   Tracing brave = Tracing.newBuilder()
-    .currentTraceContext(ThreadLocalCurrentTraceContext.newBuilder()
-      .addScopeDecorator(StrictScopeDecorator.create())
```

Comment:
Think you may have accidentally lost the strict checker

Reply:
not needed as not multi-threaded.. sorry I was cleaning up.